sdk/src/types/block/mana/mod.rs

```diff
@@ -1,7 +1,7 @@
 // Copyright 2023 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
-mod allotment;
+pub mod allotment;
```

Comment:
If ever you want to keep it consistent, we usually keep it private and re-export what's important from it